libs/component-library/src/lib/radio-button/RadioButton.tsx

```diff
@@ -136,29 +136,15 @@ const RadioButton = ({
         useNativeDriver: false,
       }).start();
     }
-  }, [checked, borderAnim, radioAnim]);
-
-  const radioBackgroundColor = disabled
-    ? rest.uncheckedDisabledBackgroundColor || colors.backgroundSecondary
-    : undefined;
-  const checkedColor = rest.checkedColor || colors.greenBase;
-  const uncheckedColor = rest.uncheckedColor || colors.border;
-  const checkedDisabledColor = rest.checkedDisabledColor || colors.greenBorder;
-  const uncheckedDisabledColor = rest.uncheckedDisabledColor || colors.border;
-
-  let radioColor: string;
-  if (disabled) {
-    radioColor = checked ? checkedDisabledColor : uncheckedDisabledColor;
-  } else {
-    radioColor = checked ? checkedColor : uncheckedColor;
-  }
+  }, [checked, borderAnim, radioAnim, isMultiSelect]);
 
   return (
     <FxPressableOpacity
       borderRadius="l"
       accessibilityRole="radio"
       accessibilityState={{ disabled, checked }}
       accessibilityLiveRegion="polite"
+      hitSlop={{ top: 0, right: 0, bottom: 0, left: 0 }}
```

Comment:
`hitSlop` can take a `number` or a [`Rect`](https://reactnative.dev/docs/rect), so you could overwrite all `Rect` from `FxPressableOpacity` with a number. I wasn't sure if this would work but tested it out in snack quick... 

This could thus be `hitSlop={0}`